.gitlab/kitchen_deploy.yml

```
@@ -122,6 +138,7 @@ deploy_deb_testing-a7_arm64:
   stage: kitchen_deploy
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
   tags: ["runner:main"]
+  <<: *deploy_deb_resource_group-a6
```

Slavek's comment that I deleted by mistake:

```suggestion
  <<: *deploy_rpm_resource_group-a6
```